lms/static/coffee/src/instructor_dashboard/data_download.coffee

```diff
@@ -159,6 +159,10 @@ class ReportDownloads
   constructor: (@$section) ->
 
     @$report_downloads_table = @$section.find ".report-downloads-table"
+    @$reports                         = @$section.find '.reports-download-container'
+    @$download_display_table          = @$reports.find '.data-display-table'
+    @$reports_request_response        = @$reports.find '.request-response'
+    @$reports_request_response_error  = @$reports.find '.request-response-error'
```

Comment:
We should avoid using whitespace to align like this. Else, you end up needing to re-align every other line each time one changes, or the formatting is destroyed, which defeats the point of doing it in the first place.